plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/StartupVspCommand.java

```diff
@@ -26,4 +26,14 @@
     public StartupVspCommand() {
         super(Host.Type.L2Networking);
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj);
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();
+    }
```

Comment:
Please implement ``toString()`` to provide debugging state information for debugging purposes.